app/src/main/java/io/github/droidkaigi/confsched/activity/ActivityNavigator.java

```diff
@@ -1,39 +0,0 @@
-package io.github.droidkaigi.confsched.activity;
-
-import android.app.Activity;
-import android.content.Context;
-import android.support.annotation.NonNull;
-import android.support.v4.app.Fragment;
-
-import javax.inject.Singleton;
-
-import io.github.droidkaigi.confsched.model.Session;
-
-@Singleton
-public class ActivityNavigator {
```

Comment:
The name and package is changed to `util/PageNavigator`.
This class is injected by Dagger. It helps us to test easily.